lib/extractor.rb

```diff
@@ -143,12 +143,12 @@ def get_extract(filepath)
       type_elem.value = "preview"
     end
 
-    unless @title_prefix.nil?
+    unless @title_prefix.nil? and @title_suffix.nil?
```

Comment:
J'ai appliqué le ruby style guide qui dit de faire du `unless something` plutôt que du `if not something`.
La condition que je veux c'est "si le suffixe _ou_ le préfixe sont non nuls".
L'inverse de (non A _ou_ non B) est (A _et_ B), d'où la condition en unless ici.

Je précise, parce que j'ai eu un peu de mal à la formuler. :D